_posts/2018-07-27-Our-first-look-at-debos-new-Debian-images-generator.md

```diff
@@ -266,12 +266,12 @@ Powering off.
 2018/07/26 19:20:44 ==== Recipe done ====
 ```
 
-So, when building for non-host architecture, we have the same error as was in
-case of `VirtualBox`. I'm not sure what might be the issue: either
-the proper `qemu-static` is not used, or some virtualization problems? Debugging
-of what's going on beneath is not easy, especially if there is no way to have
-more debug output from the `debos` or `fakemachine` tools (at least I have not
-found any).
+In a conclusion, when building for a non-host architecture we have the same
```

Comment:
Please remove "for" before "a non-host"

Reply:
@mscibisz I think It's ok. The meaning is like this one: https://docs.snapcraft.io/build-snaps/build-for-another-arch

Reply:
Not really. `Build for another architecture` means it is a build that is used to create different/another architecture.
`when building for...` + whom/what – it indicates you are performing work for someone/something.

Reply:
@mscibisz  Yes. I'm performing work for another (CPU) architecture. 
I am building (an image or a package) for another (CPU) architecture. Architecture is not created - some software package is. Is this still wrong?

Reply:
Ok then. It sounds fine and from my perspective, everything is OK with this post.